### ISSUE TITLE
Use TEXINPUTS for file search

### ISSUE THREAD

**Issue by brownzach125** [sylvainhalle/textidote#57]
textidote currently appears to only search in the root directory. Could it follow latex's behavior and use the environment variable TEXINPUTS?

**Comment by sylvainhalle** [comment]
The short answer is "yes", since it is [possible in Java to read environment variables](https://stackoverflow.com/a/20610185) --that's a note for myself. Technically speaking, it wouldn't be very complicated to add that feature.

*However*, I understand from documentation such as [this](https://www2.ph.ed.ac.uk/~wjh/tex/documents/environmental.pdf) that LaTeX has a default behavior when this variable is not defined (I just checked on my own computer and `echo $TEXINPUTS` returns the empty string). This behavior is, unfortunately, implementation-dependent and OS-dependent: it is not the same in Windows, Mac, Linux, in Texlive vs Miktex, and so on.

So, if `TEXINPUTS` is not defined, TeXtidote would need to *guess* what are the directories that the local TeX installation is looking for. You could only reliably use this feature when the user *overrides* the default behavior with their own configuration, but not when the defaults are being used. So I'd rather not support that feature at all than support it incompletely...

**Comment by brownzach125** [comment]
I agree that implementing the default behavior would be a waste of time. 
However as a user I'd rather have to recreate the default behavior by setting TEXINPUTS or passing an argument into TeXidote, than have to use absolute paths for all my includes. absolute paths are extremely inconvenient for teams working on a document.
